packages/components/src/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactNode } from "react";
+import classnames from "classnames";
+import styles from "./Drawer.css";
+import { Typography } from "../Typography";
+import { Icon } from "../Icon";
+
+interface DrawerProps {
+  readonly children: ReactNode | ReactNode[];
+  readonly title: string;
+  /**
+   * @default true
+   */
+  readonly open?: boolean;
+  onRequestClose?(): void;
```

Comment:
I think we _may_ want to make this required as if they don't hook it up the `✖️` will do nothing when clicked.
```suggestion
  onRequestClose(): void;
```